model/build.gradle

```diff
@@ -1,5 +1,33 @@
 
 description = ''
+configurations.all {
```

Comment:
As discussed & shown please keep the config in Root Build.gradle [all projects section]